### ISSUE TITLE
Dunstrc is uneditable and top padding doesn't work correctly

### ISSUE THREAD

**Issue by JuanH88** [ikz87/dots-2.0#47]
Every time I edit the dunstrc file and use the bspc wm -r command it returns to its original state, even gedit tells me that "Another program has changed the file".  I even deleted the file and ran the command, it just restores and I don't know how to undo this behavior, it also happens with lxappearance, they don't respect the changes, top padding doesn't either, although the file isn't restored, it always stays at 70 (tested even without bars)

**Comment by ikz87** [comment]
These are 2 separate problems:

dunstrc resetting has to do with #32, to quote:
> That's because I'm using templates with wpgtk, see [here](https://github.com/deviantfero/wpgtk/wiki/Templates). Basically, everytime bspwm is reset, wpgtk overwrites both `eww.yuck` and `eww.scss` with wathever is on their template, only replacing the colors so it fits the colorscheme.
> 
> To be honest, I didn't implement this very well, since I should have instead made 2 other files (something like `theme.yuck` and `theme.scss`), change the colors there, and just include those files in the bars config. Been trying to implement it that way but I've been busy.
> 
> Anyway, for now, instead of editing the file itself, you should edit the templates at `~/.config/wpg/templates/eww_mybar_eww.yuck.base` and `~/.config/wpg/templates/eww_mybar_eww.scss.base` and your changes will be applied on reload. Quick guide on templates:
> 
> ```
> {color8} will be replaced with the 8th color on your colorscheme 
> {active} will be replaced with the "active color" on your colorscheme
> 
> if you have normal braces {{
>    you will need to escape them like this
> }}
> 
> but this means you can do things like: 
> {color8}                -> #ffffff
> {color8.octal}          -> #77777777
> {active.rgb}            -> 255, 255, 255
> {userdefinedcolor.rgb}  -> 0, 0, 0
> #BA{color8.strip}       -> #BAffffff
> ```
> 
> Run `wpg` and go to the templates tab, there you will have a quick access to every template included in the repo if you wanna edit any other colorscheme dependant config files. ![image](https://user-images.githubusercontent.com/98569017/212969118-5e5b6e47-5b13-4029-b08b-2fc1b392bd7b.png)

As of now, that problem doesn't affect those files mentioned above anymore because I found a better way of integrating templates with eww's configuration, however, since dunst doesn't really provide any way to include more than one file in their config file, I'm still stuck with wpg reapplying the template to the file in each reset. 

Thank you for pointing out the padding issue, I know exactly what's causing it:
If you reset bspwm with windows already open, `eww` will reopen its windows and they will be displayed on top of all the already open ones, the problem? the top bar will still be there even if a window is fullscreen. The only acceptable fix I found for this was to check if the focused window was fullscreen after certain `bspc` events, and hide the top bar accordingly. I hide the top bar by unmapping it and setting top padding to 0, to show it again I map it and set the top padding to the hard coded value of 70. This last part is done at `~/.bscripts/toggle_bar.sh`.

TL;DR
1- For dunst, edit `~/.config/wpg/templates/config_dunst_dunstrc.base` instead of the actual `dunstrc` file

2- For the top padding, also change it in `~/.bscripts/toggle_bar.sh` at line 20 

Let me know if you manage to get things working :)

**Comment by JuanH88** [comment]
Thanks, the changes have already been applied!!!
